src/components/Pensjonsavtaler/__tests__/PensjonsavtalerMobile.test.tsx

```diff
@@ -31,11 +31,11 @@ describe('PensjonsavtalerMobile', () => {
     },
   ]
 
-  it('rendrer riktig header', async () => {
+  it('rendrer riktig header for pensjonsavtaler of offentlig tjenestepensjon', async () => {
```

Comment:
for pensjonsavtaler **of** offentlig tjenestepensjon?

Reply:
korrigert "rendrer riktig header for pensjonsavtaler og offentlig tjenestepensjon"